scripts/email/base_template.py

```diff
@@ -0,0 +1,48 @@
+import os
+from dotenv import load_dotenv
+import resend
+from typing import List
+
+# Load relevant template to run this script on.
+from templates.dryrun import *
+
+# Load environment variables from .env
+load_dotenv(".env")
+
+resend.api_key = os.environ["RESEND_API_KEY"]
+
+# Global constants across all sessions — please do not change these.
+FROM = os.environ["GLOBAL_FROM_NAME"] + " <" + os.environ["GLOBAL_FROM_EMAIL"] + ">"
+TO = os.environ["GLOBAL_TO_EMAIL"]
+
+
+def chunk_list(lst: List[str], chunk_size: int) -> List[List[str]]:
+    return [lst[i : i + chunk_size] for i in range(0, len(lst), chunk_size)]
+
+
+def send_emails(bcc_list: List[str]):
+    bcc_chunks = chunk_list(bcc_list, 49)
+    total_chunks = len(bcc_chunks)
+
+    print(
+        f"Starting email job with {len(bcc_list)} recipients in {total_chunks} batch(es)."
+    )
```

Comment:
The checks per batch are super helpful! anyway to add a check incase an email is not successfully sent to _one_ specific user in a batch? We should look at some external notification / alert (outside of the console / terminal) that devs get if an email is not successfully sent, especially in the future if we want email notifications to be more regular

Reply:
Hmm yes definitely, let's talk about how this might look at the next dev sesh? Sorry for delay in getting back to you on this.